daemon/containerd/image_builder.go

```diff
@@ -460,11 +460,15 @@ func (i *ImageService) CreateImage(ctx context.Context, config []byte, parent st
 
 	// necessary to prevent the contents from being GC'd
 	// between writing them here and creating an image
-	ctx, done, err := i.client.WithLease(ctx, leases.WithRandomID(), leases.WithExpiration(1*time.Hour))
+	ctx, release, err := i.client.WithLease(ctx, leases.WithRandomID(), leases.WithExpiration(1*time.Hour))
 	if err != nil {
 		return nil, err
 	}
-	defer done(ctx)
+	defer func() {
+		if err := release(ctx); err != nil && !cerrdefs.IsNotFound(err) {
```

Comment:
If we are going to log this then we shouldn't filter the not found error since this is something that shouldn't happen because this is a lease that we created

Reply:
Yeah I would prefer if we logged them regardless of the error

Reply:
Alright; let me update 👍